artemis-prometheus-metrics-plugin-servlet/pom.xml

```diff
@@ -24,7 +24,7 @@ under the License.
    <parent>
       <groupId>com.redhat.amq-broker</groupId>
       <artifactId>artemis-prometheus-metrics-pom</artifactId>
-      <version>2.0.0</version>
+      <version>2.0.1</version>
```

Comment:
Could you move the version change to separate commit, i.e. https://github.com/rh-messaging/artemis-prometheus-metrics-plugin/commit/d0a9b3ea0e911d2cbeb89fb64d0b27bc240b55ea

Reply:
I'll push a commit to bump the version when your fix will be merged.